config/set/php80.php

```diff
@@ -58,7 +55,6 @@
             ArgumentAdderRector::class,
             [new ArgumentAdder('Nette\Utils\Strings', 'replace', 2, 'replacement', '')]
         );
-    $rectorConfig->rule(RemoveParentCallWithoutParentRector::class);
```

Comment:
if you found a bug on `RemoveParentCallWithoutParentRector`, the patch should be in the rule instead.